CLAUDE.md

```
@@ -0,0 +1,107 @@
+- `updated: "auto"` - 自动获取更新日期
+- 生产环境自动过滤 `test` 分类和 `test` 前缀文章
+
+### 特殊分类
+
+定义在 `src/constants/categories.ts`:
+- `CATEGORY_ALL = "all"` - 虚拟分类，路由为 `/blog` 而非 `/blog/category/all`
+- `CATEGORY_ESSAY = "essay"` - 随笔，使用时间轴布局，在首页最新文章中排除
+
+### 路由结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)